util/cron/nightly

```diff
@@ -629,7 +630,7 @@ if ($examples == 1) {
     $testdirs .= " release/examples";
 }
 
-if ($blogonly == 1) {
+if ($blogonly == 1 || ($testblog == 1 && $examples == 1)) {
```

Comment:
So `-examples -blog` means test only the blog examples, rather than test both the blog and the other examples? That seems counterintuitive to me

Reply:
hmm, no, it should test _both_ the blog and examples (and my local testing confirmed it did), as `$testdirs .= ...` is like `+=`

with `-blog -examples`
```
[Summary: #Successes = 438 | #Failures = 6 | #Futures = 0 | #Warnings = 0 ]
```

with just `-examples`
```
[Summary: #Successes = 399 | #Failures = 0 | #Futures = 0 | #Warnings = 0 ]
```

Reply:
Hmm, then does `-blogonly` not mean test only the blog examples? Since this same block of code implements it and does not exclude things besides the blog? I don't understand the difference between the `-blog` and `-blogonly` flags.